packages/devtools_app_shared/example/utils_example.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app_shared/utils.dart';
+import 'package:flutter/widgets.dart';
+
+void main() {
+  /// Example: setting and accessing globals.
+  setAndAccessGlobal();
+
+  /// Example: [ListValueNotifier]
+  useListValueNotifier();
+}
+
+MyCoolClass get coolClass => globals[MyCoolClass] as MyCoolClass;
+
+void setAndAccessGlobal() {
```

Comment:
Consider adding some documentation to each of the use cases in this file describing:

- What functionality is being exercised
- Why it's available
- When a developer might want to use it

I'd also consider keeping each of these examples in their own self-contained libraries to keep things as simple as possible. Ideally each example could be executed standalone and clearly demonstrate the expected behaviors of the APIs.

Reply:
I split up the utils examples into individual files. Left the ui and service as they are.